test/test-creation.js

```diff
@@ -98,7 +98,7 @@ describe('react-webpack generator', function() {
       react.run({}, function() {
         assert.fileContent([
             ['package.json', /react-hot-loader/],
-            ['Gruntfile.js', /hot:\s*true/],
+            ['Gruntfile.coffee', /hot:\s*true/],
```

Comment:
With coffee, `hot: true` may be replaced to `hot: yes` or `hot: no`. This check is becoming *unsafe*.